### PR TITLE
Feature/join section and page

### PR DESCRIPTION
**Pull request by RitikJaiswal75**
### Issue #274 
Created a section join page for main site.

### What is the change?
- created a new file join.html
- created a new file feature-flag.js
- modified main.css
- modified index.html

### How to access join section
pass a parameter in URL : `/?join=true` to access the join section

### Is Development Tested?

- [x] Yes
- [ ] No

https://user-images.githubusercontent.com/57758447/187619935-119aa31f-ac60-4fd5-bd1c-ac6b23b0eb76.mp4

### CODE REVIEW

File: join.html [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+      </ul>
+    </div>
+    <button id="scroll-to-top-btn">
+      <img src="./img/arrow_up.svg" alt="scroll to top button" />
+    </button>
+    <nav>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
```
Comment (swarajpure) [inline join.html:90]: Could these be self closing? `<span class="bar" />`

File: join.html [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    <main>
+      <section class="section-join">
+        <h1 class="features-title">How to join Real Dev Squad</h1>
+        <div class="join-description">
+          <p>
+            Follow Ankush on twitter, Message him, he will send you a link to
+            join
+          </p>
+        </div>
+      </section>
```
Comment (swarajpure) [inline join.html:163]: And how are we redirecting people to Ankush's twitter account? Shouldn't we add a hyperlink here to his twitter account?

File: join.html (hunk 1 above)
Comment (RitikJaiswal75) [reply join.html:90]: I tried to keep the code structure same as the one in Index.html

File: join.html (hunk 2 above)
Comment (RitikJaiswal75) [reply join.html:163]: I did not add any hyperlinks, Even the content was not decided so not sure this should be done its just a feature flag, once the content is fixed it will change

File: index.html [hunk 3]
```diff
@@ -237,6 +237,22 @@ <h1 class="description-title">What is Real Dev Squad?</h1>
         </div>
       </section>
 
+      <section class="section-features hide" id="section-join">
+        <h1 class="features-title">How to Join RealDev Squad</h1>
```
Comment (rohan09-raj) [inline index.html:241]: Typo, put space between `Real` and `Dev`

File: index.html [hunk 4]
```diff
@@ -237,6 +237,22 @@ <h1 class="description-title">What is Real Dev Squad?</h1>
...
+        <h1 class="features-title">How to Join Real Dev Squad</h1>
+        <div class="join-description">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorem
+            doloribus repudiandae aliquam blanditiis molestias impedit
+            voluptates qui deleniti laboriosam nesciunt? Quas ullam consequuntur
+            assumenda molestiae dolorem ut iusto pariatur corporis porro harum
+            accusamus sint veniam et dignissimos asperiores, id velit expedita
+            officiis labore libero possimus. Incidunt suscipit esse vero
+            provident.
```
Comment (rohan09-raj) [inline index.html:250]: Can we please add the actual content ??

File: join.html [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+    <meta name="title" content="Real Dev Squad" />
+    <meta
+      name="description"
+      content="Community of developers who collaborate to build Real software."
+    />
+
+    <!-- Open Graph / Facebook Meta Tags -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://www.realdevsquad.com/" />
+    <meta property="og:title" content="Real Dev Squad" />
```
Comment (rohan09-raj) [inline join.html:27]:
```suggestion
    <meta property="og:title" content="Real Dev Squad | Join" />
```
Tiltle can be as above, as it's a different page

File: join.html [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+    <!-- Primary Meta Tags -->
+    <meta name="title" content="Real Dev Squad" />
+    <meta
+      name="description"
+      content="Community of developers who collaborate to build Real software."
+    />
+
+    <!-- Open Graph / Facebook Meta Tags -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://www.realdevsquad.com/" />
```
Comment (rohan09-raj) [inline join.html:26]: Please add the relevant URL to this page

File: join.html [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+    <meta property="og:title" content="Real Dev Squad" />
+    <meta
+      property="og:description"
+      content="Community of developers who collaborate to build Real software."
```
Comment (rohan09-raj) [inline join.html:30]: Description can also be changed according to the page

File: join.html [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+    />
+
+    <!-- Twitter Meta Tag-->
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:site" content="@RealDevSquad" />
+    <meta property="twitter:title" content="Real Dev Squad" />
+    <meta
+      property="twitter:description"
+      content="Community of developers who collaborate to build Real software."
+    />
```
Comment (rohan09-raj) [inline join.html:44]: Can we please change it according to the changes to be done in Primary Meta Tags ?

**Review by rohan09-raj — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (vinayak-trivedi) [inline index.html:250]: why is this here?

Comment (RitikJaiswal75) [reply index.html:250]: Sorry Rohan but the content was not fixed I asked Ankush but he told me to keep it hidden. Can I get what content to put?

Comment (RitikJaiswal75) [reply index.html:250]: Since the content wasn't provided to me.

File: js/navbar.js [hunk 9]
```diff
@@ -24,3 +24,16 @@ document.querySelectorAll('.user-greet').forEach((greet) => {
...
+
+document.addEventListener('click', (event) => {
+  if (
+    dropdownTrigger[0] != event.target.parentElement &&
+    dropdownTrigger[1] != event.target.parentElement &&
+    !document.querySelector('.dropdown').classList.contains('hide')
+  ) {
+    document.querySelector('.dropdown').classList.add('hide');
+  }
+});
```
Comment (rohan09-raj) [inline js/navbar.js:39]: Can we please remove it from here as it is already addressed in separate PR ?
https://github.com/Real-Dev-Squad/website-www/pull/276/files

File: index.html (hunk 4 above)
Comment (rohan09-raj) [reply index.html:250]: Ok let's leave it for now, it is behind the feature flag right

**Review by rohan09-raj — COMMENTED** [review]
@RitikJaiswal75 Can we please center align the contents as in the video I can see the content is not aligned or if you have already done that can you please update the video? 
Also can we please put h2 heading of `under development-more details to be added soon` in red color bold text?

**Comment by RitikJaiswal75** [comment]
> @RitikJaiswal75 Can we please center align and equally give font size to the contents as in the video I can see the content is not aligned and font is too small for content? Also can we please put a h2 heading of `under development-more content to be added soon` in red color bold text? In that way, some other developer will know right away that it's a under development feature

I have made the asked changes.

Latest screenshot:
image

**Review by rohan09-raj — APPROVED** [review]
